src/tests/radmin/all.mk

```diff
@@ -54,6 +54,5 @@ $(OUTPUT)/%: $(DIR)/% test.radmin.radiusd_kill test.radmin.radiusd_start
 		touch $@;\
 	fi
 
-$(TEST): $(FILES)
+$(TEST): ${BUILD_DIR}/bin/radmin $(FILES)
```

Comment:
That should just edit the build rules, like src/tests/unit/all.mk:

```
#
#	Run the radmin commands against the radiusd.
#
$(OUTPUT)/%: ${BUILD_DIR}/bin/radmin $(DIR)/% test.radmin.radiusd_kill test.radmin.radiusd_start
```

Reply:
done